doc/developers/contributing.rst

```diff
@@ -14,6 +14,26 @@ adding new algorithms, and the best way to contribute and to help the project
 is to start working on known issues.
 See :ref:`easy_issues` to get started.
 
+.. topic:: **Our community, our values**
+
+    We are a community based on openness and friendly, didactic,
+    discussions.
+
+    We aspire to treat everybody equally, and value their contributions.
+
+    Decisions are made based on technical merit and consensus.
```

Comment:
Don't the above three lines go without saying? It can kind of imply that in general, these things are not followed. My views are similar to those over here (https://github.com/vinta/awesome-python/issues/536#issuecomment-166115107) but maybe not the same language.

In comparison. the next paragraph is more useful since it states the importance of stuff apart from explicit commits (which is generally undervalued)

I have no strong objection in merging this, just letting you know my views.